src/v3/webpack.testcafe.config.ts

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
This is the same as the dev config, the only difference being the duo package is using mocks instead of the real module.